config/project/entryTypes/resource--992c932b-21e6-4b0b-8017-1289891c1ac4.yaml

```diff
@@ -0,0 +1,290 @@
+color: null
+fieldLayouts:
+  227e9d26-0f8f-44ef-b668-c439eff5df46:
+    tabs:
+      -
+        elementCondition: null
+        elements:
+          -
+            autocapitalize: true
+            autocomplete: false
+            autocorrect: true
+            class: null
+            dateAdded: '2025-02-27T04:32:12+00:00'
+            disabled: false
+            elementCondition: null
+            id: null
+            includeInCards: false
+            inputType: null
+            instructions: null
+            label: null
+            max: null
+            min: null
+            name: null
+            orientation: null
+            placeholder: null
+            providesThumbs: false
+            readonly: false
+            required: true
+            size: null
+            step: null
+            tip: null
+            title: null
+            type: craft\fieldlayoutelements\entries\EntryTitleField
+            uid: b3fabcf7-360b-455c-8563-c85ab83e64bb
+            userCondition: null
+            warning: null
+            width: 100
+          -
+            dateAdded: '2025-02-28T08:54:00+00:00'
+            elementCondition: null
+            fieldUid: 69339afc-d4f1-440b-a7ad-7da2ccb77972 # Image
+            handle: cover
+            includeInCards: false
+            instructions: null
+            label: 'Cover Image'
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 5e08372c-a233-407f-8e58-9e3782a8612b
+            userCondition: null
+            warning: null
+            width: 100
+          -
+            dateAdded: '2025-02-28T08:54:00+00:00'
+            elementCondition: null
+            heading: 'Document Data'
+            type: craft\fieldlayoutelements\Heading
+            uid: d30e5818-039e-4fba-8c68-19e1dacf53a0
+            userCondition: null
+          -
+            dateAdded: '2025-02-28T08:54:00+00:00'
+            elementCondition: null
+            fieldUid: 32ed2f1a-9024-4ee9-9028-3a5041c0e5e6 # Date
+            handle: null
+            includeInCards: false
+            instructions: null
+            label: 'Date Updated'
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: e9976c4e-75b9-4b12-bc2d-5f9e8cf8ca6e
+            userCondition: null
+            warning: null
+            width: 100
+          -
+            dateAdded: '2025-02-27T07:24:28+00:00'
+            elementCondition: null
+            fieldUid: 1a28c01c-3fa0-4a60-8c73-d7c7707be626 # Document
```

Comment:
Move _Document_ to the same tab as Body to reinforce the connection between the two and communicate to authors that they can either add native HTML content or an asset, or both, but that the asset field isn’t mandatory.